<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Start</class>
 <widget class="QMainWindow" name="Start">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>805</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Start</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="mainWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>500</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page_login">
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>280</y>
        <width>320</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_name">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>200</y>
        <width>45</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:15px</string>
      </property>
      <property name="text">
       <string>用户名</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_password">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>260</y>
        <width>30</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:15px
</string>
      </property>
      <property name="text">
       <string>密码</string>
      </property>
     </widget>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>288</x>
        <y>9</y>
        <width>200</width>
        <height>200</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>200</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border-image : url(:/resource/产品logo-02.png)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>220</y>
        <width>320</width>
        <height>30</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>30</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
     </widget>
     <widget class="QPushButton" name="button_login">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>360</y>
        <width>100</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : rgb(39, 199, 183);
font-family : &quot;Microsoft YaHei&quot;;
font-size : 15px;
color : #FFFFFF;
border : none;</string>
      </property>
      <property name="text">
       <string>登 录</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>400</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family : &quot;Microsoft YaHei&quot;;
font-size : 15px;
color : #27C7B7;
background-color : rgba(255, 255, 255, 0);
outline: none;</string>
      </property>
      <property name="text">
       <string>注册一个用户</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_login_fault">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>310</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:15px;
color : #dd0000;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_logined">
     <widget class="QWidget" name="widget_user_info" native="true">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>0</y>
        <width>200</width>
        <height>500</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color : #27C7B7</string>
      </property>
      <widget class="QGraphicsView" name="graphicsView_face">
       <property name="geometry">
        <rect>
         <x>52</x>
         <y>50</y>
         <width>96</width>
         <height>96</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border : none;</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_user_name">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>170</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">color : #FFFFFF;
font-family: &quot;Microsoft YaHei&quot;;
font-size:21px;</string>
       </property>
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>200</y>
         <width>181</width>
         <height>21</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border-color : #FFFFFF;
</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_xp">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>219</y>
         <width>180</width>
         <height>51</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:21px;
color : #FFFFFF;</string>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_progress">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>160</width>
         <height>8</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border : none;
background-color : #5DD5C9;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_progress_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>93</width>
         <height>8</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">border : none;
background-color : #AEEAE4;
</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QStackedWidget" name="widget_problem_info">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>600</width>
        <height>500</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page_select">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>55</y>
          <width>480</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:21px;
color : #666666;</string>
        </property>
        <property name="text">
         <string>流行传染病知识题库 ————————————————————</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_problem_set_2">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>290</y>
          <width>461</width>
          <height>171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>图标</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:21px</string>
        </property>
        <property name="text">
         <string>题库</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_problem_set">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>80</y>
          <width>461</width>
          <height>171</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="columnCount">
         <number>1</number>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderShowSortIndicator" stdset="0">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
        <column>
         <property name="text">
          <string>图标</string>
         </property>
        </column>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>260</y>
          <width>481</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:21px;
color : #666666;</string>
        </property>
        <property name="text">
         <string>收藏夹————————————————————————</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_question">
       <widget class="QLabel" name="label_title">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>40</y>
          <width>161</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:21px</string>
        </property>
        <property name="text">
         <string>题库名字</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_progress">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>80</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:21px;
color : #27C7B7;
</string>
        </property>
        <property name="text">
         <string>进度</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_question">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>120</y>
          <width>501</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family: &quot;Microsoft YaHei&quot;;
font-size:15px;
color : #333333;</string>
        </property>
        <property name="text">
         <string>题库</string>
        </property>
       </widget>
       <widget class="Button_answer" name="button_answer_0">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>180</y>
          <width>480</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="Button_answer" name="button_answer_1">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>220</y>
          <width>480</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="Button_answer" name="button_answer_2">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>260</y>
          <width>480</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="Button_answer" name="button_answer_3">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>300</y>
          <width>480</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_confirm">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>420</y>
          <width>80</width>
          <height>30</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color : #E6E6E6;
font-family : &quot;Microsoft YaHei&quot;;
font-size : 15px;
color : #B3B3B3;
border : none;</string>
        </property>
        <property name="text">
         <string>确 认</string>
        </property>
       </widget>
       <widget class="QPushButton" name="button_return">
        <property name="geometry">
         <rect>
          <x>60</x>
          <y>420</y>
          <width>121</width>
          <height>31</height>
         </rect>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">font-family : &quot;Microsoft YaHei&quot;;
font-size : 15px;
color : #27C7B7;
background-color : rgba(255, 255, 255, 0);
outline: none;</string>
        </property>
        <property name="text">
         <string>返 回</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Button_answer</class>
   <extends>QPushButton</extends>
   <header location="global">button_answer.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
